src/cclib/parser/adfparser.py

```diff
@@ -136,10 +139,20 @@ def extract(self, inputfile, line):
             while line[:5] != "title":
                 line = inputfile.next()
 
+        if "Amsterdam Density Functional" in line:
+            self.metadata['package_version'] = line.split()[5]
+
         if line[1:10] == "Symmetry:":
             info = line.split()
             if info[1] == "NOSYM":
                 self.nosymflag = True
+                point_group_full = "c1"
+            else:
+                point_group_full = info[1].replace('(', '').replace(')', '').lower()
+            # TODO
+            point_group_abelian = point_group_full
+            self.metadata['symmetry_full'] = point_group_full
+            self.metadata['symmetry_abelian'] = point_group_abelian

```

Comment:
Doing it this way shows both

1. what the maximum point group detected was (`full`), and
2. what the calculation was actually run in (`abelian`).

Take benzene for example. If these are set to `"d6h"` and `False`, it doesn't tell you if the calculation was run in `"d2h"`, `"c2v"`, `"c1"`, etc.

If you choose to have `symmetry_full` be the point group the calculation was run in, and not the full molecular point group, these will be set to `"d2h"` and `True`. You can have benzene or something like it _actually_ be D2h, so some information has been lost.

Pinging @ghutchis to see what he'd like implemented.